src/graph/backend/dnnl/kernels/sdp_primitive.cpp

```diff
@@ -272,6 +291,7 @@ status_t sdp_primitive_kernel_t<quantized>::sycl_execute_impl(
     sycl_stream->after_exec_hook();
 
     return status;
+#endif
```

Comment:
what is this `#endif` for?

Reply:
It's the `#endif` for the `#if DNNL_GPU_VENDOR != DNNL_VENDOR_INTEL` at L250. Do you see any mismatching?